cmd/juju/commands/repl.go

```diff
@@ -97,8 +96,12 @@ func (c *replCommand) Run(ctx *cmd.Context) error {
 		if err := jujuCmd.Init([]string{"help"}); err != nil {
 			return errors.Trace(err)
 		}
+
+		if err := jujuCmd.Run(ctx); err != nil {
+			return err
+		}
 		fmt.Fprintln(ctx.Stdout, replDoc)
```

Comment:
This ensures that the REPL output appears **below** the summarized help text (see QA steps output)